wicket-core/src/main/java/org/apache/wicket/mock/MockApplication.java

```diff
@@ -71,5 +71,7 @@ protected void internalInit()
 
 		// for test cases we usually want stable resource names
 		getResourceSettings().setCachingStrategy(NoOpResourceCachingStrategy.INSTANCE);
+		// the core CSS causes noise in tests and isn't needed
+		getResourceSettings().disableWicketCoreCSS();
```

Comment:
Could you please expand this comment explaining what kind of noise.

Reply:
The "noise" is the extra `<link>` element in the `<head>` of every `**-expected.html` in the test suite